tests/virt_autotest/login_console.pm

```diff
@@ -224,6 +224,21 @@ sub login_to_console {
     set_ssh_console_timeout_before_use if (is_sle and is_remote_backend and is_x86_64 and get_var('VIRT_AUTOTEST', ''));
     # use console based on ssh to avoid unstable ipmi
     use_ssh_serial_console;
+
+    # Check 64kb page size enabled.
+    if (check_var('KERNEL_64KB_PAGE_SIZE', '64kpg')) {
```

Comment:
I think this should be a boolean setting ?

Reply:
Changed KERNEL_64KB_PAGE_SIZE=1